gensim/utils.py

```diff
@@ -1260,12 +1259,12 @@ def chunkize(corpus, chunksize, maxsize=0, as_numpy=False):
         maxsize : int, optional
             If > 0, prepare chunks in a background process, filling a chunk queue of size at most `maxsize`.
         as_numpy : bool, optional
-            Yield chunks as `np.ndarray`s instead of lists?
+            Yield chunks as `np.ndarray` instead of lists?
 
         Yields
         ------
         list OR np.ndarray
-            `chunksize`-ed chunks of elements from `corpus`.
+            "chunksize"-ed chunks of elements from `corpus`.
```

Comment:
Why this change? `chunksize` is a parameter name (just like ``corpus``, not `"corpus"`).

Reply:
in Sphinx construction
```
`something`other
```
without space between "`" and "o" break render